From 00fc9418ec62060aeba4d101ee59b4c8dce7c82b Mon Sep 17 00:00:00 2001
From: Indhi Rousseau <contact@thekinrar.fr>
Date: Thu, 23 Apr 2020 02:59:58 +0200
Subject: [PATCH] Remove version from server brand

This was unreadable, most likely useless and did not match Papyrus behaviour

diff --git a/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java b/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java
index d7d0fafc..a9c14573 100644
--- a/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java
+++ b/proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java
@@ -281,7 +281,7 @@ public class DownstreamBridge extends PacketHandler
             Preconditions.checkState( !serverBrand.contains( bungee.getName() ), "Cannot connect proxy to itself!" );
 
             brand = ByteBufAllocator.DEFAULT.heapBuffer();
-            DefinedPacket.writeString( bungee.getName() + " (" + bungee.getVersion() + ")" + " <- " + serverBrand, brand );
+            DefinedPacket.writeString( bungee.getName() + " <- " + serverBrand, brand );
             pluginMessage.setData( brand );
             brand.release();
             // changes in the packet are ignored so we need to send it manually
-- 
2.26.1

